src/server/services/generate-arpa-report.js

```diff
@@ -81,7 +87,64 @@ async function generateProject519521 (periodId) {
 }
 
 async function generateProjectBaseline (periodId) {
-  return loadTemplate('Project Templates/projectBaselineBulkUpload')
+  const aoa = loadTemplate('Project Templates/projectBaselineBulkUpload')
+  console.log(aoa)
+
+  let documents
+  try {
+    documents = await documentsWithProjectCode(periodId)
+  } catch (_err) {
+    return new Error('Failed to load document records')
+  }
+
+  const rowObject = {}
+
+  documents.forEach(document => {
+    console.log(document)
+
+    switch (document.type) {
+      case 'certification':
+        break
+      case 'cover':
+        rowObject.Project_Expenditure_Category__c = document.content['expenditure category group']
+        rowObject.Sub_Category__c = document.content['detailed expenditure category']
+    }
```

Comment:
instead of these switch statements, can we `filter`? or possibly even ask the `documentsWithProjectCode` for only the types of documents we care about?